R/google_slides.R

```diff
@@ -0,0 +1,229 @@
+#' Get Slide ID from URL
```

Comment:
This is not new code I just put all the stuff pertaining to google slides in one spot.